tests/code/test_userManagement.py

```diff
@@ -147,20 +147,9 @@ def test_changeProgramInfo():
 @pytest.mark.integration
 def test_updatedProgramManager():
     with mainDB.atomic() as transaction:
-        # Try to add a student who isnt Student Staff into a Program Manager: They should not be added.
+        # Make student a program manager.
```

Comment:
I am kind of nitpicky about testing but I feel like you should make a new user that is specifically not a student staff and try this with a couple different roles. It isn't great for testing visibility to just have a user imported from test data we don't know anything about.